### PR TITLE
Adds a new admin smite, Dock Pay.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

image
Admins can now smite players with an ID on by docking their pay, enabling them to remove money from a player's account remotely and swiftly. Fails if they don't have an ID on, or has on an ID that doesn't have an account.

## Why It's Good For The Game

More smites -> More ways to discourage bad behavior/dick about.

## Changelog
:cl:
add: Your sector has upgraded the BSA with a new specialty lens, Remote Economic Punishment Bombardment.
/:cl:

### CODE REVIEW

**Comment by lordpidey** [comment]
Change the last word in the change log to "Ordinance"

**R**emote **E**conomic **P**unishment **O**rdinance

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -1238,6 +1239,29 @@ Traitors and the like can also be revived with the previous role mostly intact.
...
+				to_chat(usr,"<span class='warning'>[dude] does not have an ID card on!</span>", confidential = TRUE)
+				return
+			if(!card.registered_account)
+				to_chat(usr,"<span class='warning'>[dude] does not have an ID card with an account!</span>", confidential = TRUE)
+				return
+			var/new_cost = input("How much pay are we docking?","BUDGET CUTS") as num|null
+			if(!new_cost)
+				return
+			if(!(card.registered_account.has_money(new_cost)))
+				to_chat(usr, "<span class='warning'>ID Card lacked funds. Emptying account.</span>")
```
Comment (Rohesie) [inline code/modules/admin/verbs/randomverbs.dm:1258]:
```suggestion
				return
			if(card.registered_account.account_balance == 0)
				to_chat(usr, "<span class='warning'>ID Card lacks any funds. No pay to dock.</span>")
				return
			var/new_cost = input("How much pay are we docking? Current balance: [card.registered_account.account_balance] credits.","BUDGET CUTS") as num|null
			if(!new_cost)
				return
			if(!(card.registered_account.has_money(new_cost)))
				to_chat(usr, "<span class='warning'>ID Card lacked funds. Emptying account.</span>")
```

**Comment by optimumtact** [comment]
is there a limit to how many smites there are.

**Review by Rohesie — APPROVED** [review]
Code checks.

**Pull request merged**